VM/inventory

```diff
@@ -0,0 +1,2 @@
+[ec2Instances]
+ec2-18-217-80-152.us-east-2.compute.amazonaws.com
```

Comment:
I copy and pasted the instance public address. But that doesn't feel right at all.